be/src/util/mem_info.h

```diff
@@ -78,11 +82,13 @@ class MemInfo {
     static int64_t _s_mem_limit;
     static int64_t _s_hard_mem_limit;
     static size_t _s_allocator_physical_mem;
+    static size_t _s_pageheap_unmapped_bytes;
```

Comment:
please update document
`docs/zh-CN/docs/admin-manual/maint-monitor/monitor-metrics/metrics.md`

Reply:
These detailed properties of tcmalloc are not added to metrics,
I will update memory related metrics later.